Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -901,6 +901,19 @@
     <!-- XHED: Title for native sharing dialog -->
     <string name="debugging_debuglog_sharing_dialog_title">"CWA Fehlerbericht teilen"</string>
 
+    <!-- XHED: Title for debug legal screen -->
+    <string name="debugging_debuglog_legal_dialog_title">"Ausführliche Informationen zur Übersendung der Fehlerberichte"</string>
```

Comment:
@harambasicluka  @janetback  Do these need to go into the `legal_strings.xml`?

Reply:
Previously I have to add title to `legal_strings.xml` too.

Reply:
is changed